src/vsr/replica.zig

```diff
@@ -4797,6 +4801,10 @@ pub fn ReplicaType(
                     }
 
                     if (self.status == .recovering_head) {
+                        if (message.header.view < self.superblock.working.vsr_state.checkpoint.header.view) {
```

Comment:
Add a `log.debug("... ignoring (...)")` line. (The "ignoring" debug lines are some of my favorites for troubleshooting in the vopr :smile:).